utils/src/main/kotlin/Utils.kt

```diff
@@ -33,6 +33,11 @@ val log = org.slf4j.LoggerFactory.getLogger({}.javaClass) as ch.qos.logback.clas
  */
 var printStackTrace = false
 
+/**
+ * The name of the ORT configuration file.
+ */
+const val ORT_CONFIG_FILE = ".ort.yml"
```

Comment:
Would it make sense to change this to

ORT_CONFIG_FILENAME_EXTENSION = ".ort.yml"

to indicate that this is not necessarily the actual file name but that we expect the filename to have this suffix?!

Reply:
I'd maybe say to use `ORT_CONFIG_FILE_NAME` (or `ORT_CONFIG_FILENAME`) then, because in most cases it actually is the full name, not only the suffix.

Reply:
I don't really mind whether the var is called ORT_CONFIG_FILE, ORT_CONFIG_FILE_NAME or ORT_CONFIG_FILENAME, seems all ok to me.

Reply:
I still decided to go for `ORT_CONFIG_FILENAME`.